.github/workflows/ci.yaml

```diff
@@ -8,59 +8,56 @@ on:
 
 jobs:
 
-  lint:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938  # v4.2.0
-        with:
-          fetch-depth: 0 # full history for setuptools_scm
-
-      - name: Set up Python
-        uses: actions/setup-python@f677139bbe7f9c59b41e40162b753c062f5d49a3  # v5.2.0
-        with:
-          python-version: 3.9
-
-      - name: Run pre-commit
-        uses: pre-commit/action@2c7b3805fd2a0fd8c1884dcaebf91fc102a13ecd  # v3.0.1
-
-  typing:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938  # v4.2.0
-        with:
-          fetch-depth: 0 # full history for setuptools_scm
-
-      - name: Set up Python
-        uses: actions/setup-python@f677139bbe7f9c59b41e40162b753c062f5d49a3  # v5.2.0
-        with:
-          python-version: 3.9
-
-      - name: Python install
-        run: |
-          python -m pip install --upgrade pip
-          pip install tox
-
-      - name: Cache tox environments
-        id: cache-tox
-        uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9  # v4.0.2
-        with:
-          path: .tox
-          key: tox-typing-${{ hashFiles('setup.cfg') }}-${{ hashFiles('pyproject.toml') }}-${{ hashFiles('tox.ini') }}
-
-      - name: Run mypy
-        run: tox -e typing
+  # lint:
```

Comment:
Why commented instead of deleted?

Reply:
Just in case someone later wants to re-implement without going through the commit history, but I can delete it if you'd prefer.

Reply:
Hmm how about just a `if: false` with comment? The diff would at least be smaller.